packages/loader/src/gltf/extensions/MeshoptDecoder.ts

```diff
@@ -106,39 +139,39 @@ const MeshoptDecoder = (function () {
       ".then(function(result) {instance = result.instance; instance.exports.__wasm_call_ctors();});\n" +
```

Comment:
`   function initWorkers(count) {
       const script = generateWorkerScript();
       const blob = new Blob([script], { type: "text/javascript" });
       const url = URL.createObjectURL(blob);
       for (let i = 0; i < count; ++i) {
           workers[i] = createWorker(url);
       }
       URL.revokeObjectURL(url);
   }

   function generateWorkerScript() {
       return `
       var instance;
       var ready = WebAssembly.instantiate(new Uint8Array([${new Uint8Array(unpack(wasm)).toString()}]), {})
           .then(result => {
               instance = result.instance;
               instance.exports.__wasm_call_ctors();
           });
       self.onmessage = async function(event) {
           await ready;
           const data = event.data;
           try {
               const target = new Uint8Array(data.count * data.size);
               await decode(instance.exports[data.mode], target, data.count, data.size, data.source, instance.exports[data.filter]);
               self.postMessage({ id: data.id, count: data.count, action: "resolve", value: target }, [target.buffer]);
           } catch (error) {
               self.postMessage({ id: data.id, count: data.count, action: "reject", value: error });
           }
       };
       `;
   }
`